rdwatch/core/db/functions.py

```diff
@@ -78,6 +78,12 @@ def __init__(self, field):
         return super().__init__(json_str, JSONField())  # noqa: B037
 
 
+class AsGeoJSONDeserialized(Cast):
```

Comment:
If there's an easier way of getting `AsGeoJSON`'s result into a deserialized python object, I'd love to know it.